cypress/integration/v2/blocks/icon-list.spec.js

```diff
@@ -56,6 +57,17 @@ function switchDesign() {
 	] ) )
 }
 
+function typeContent() {
```

Comment:
Icon List doesn't use register block snapshots because it uses a core block for text. It has to use default gutenberg assertBlockContent behavior

Reply:
Icon List doesn't use core block for text.

Reply:
When I inspected the text of icon list, I found that it has a class of "block-editor-rich-text__editable." I couldn't check the text in the frontend because this class disappears once it is published

Reply:
The `block-editor-rich-text__editable` class is used by Gutenberg for most editable content inside blocks. It's only used within Gutenberg and it's not saved with the block.

Reply:
Ohhh I see! Thank you!